tools/describe_collection.py

```diff
@@ -0,0 +1,60 @@
+#! /home/nsedaghat/.conda/envs/nimas/bin/python3
```

Comment:
Instead of hard-coding this path, could you instead write this script as a python function?